src/transport/SessionManager.cpp

```diff
@@ -310,25 +335,20 @@ CHIP_ERROR SessionManager::SendPreparedMessage(const SessionHandle & sessionHand
 
         destination = &secure->GetPeerAddress();
 
-        ChipLogProgress(Inet,
-                        "Sending %s msg %p with MessageCounter:" ChipLogFormatMessageCounter " to 0x" ChipLogFormatX64
-                        " (%u) at monotonic time: " ChipLogFormatX64 " msec",
-                        "encrypted", &preparedMessage, preparedMessage.GetMessageCounter(),
-                        ChipLogValueX64(secure->GetPeerNodeId()), secure->GetFabricIndex(),
-                        ChipLogValueX64(System::SystemClock().GetMonotonicMilliseconds64().count()));
+        ChipLogProgress(Inet, "(S) Sending msg " ChipLogFormatMessageCounter " on secure session with LSID: %u",
+                        preparedMessage.GetMessageCounter(), secure->GetLocalSessionId());
     }
     break;
     case Transport::Session::SessionType::kUnauthenticated: {
         auto unauthenticated = sessionHandle->AsUnauthenticatedSession();
         unauthenticated->MarkActive();
         destination = &unauthenticated->GetPeerAddress();
 
-        ChipLogProgress(Inet,
-                        "Sending %s msg %p with MessageCounter:" ChipLogFormatMessageCounter " to 0x" ChipLogFormatX64
-                        " at monotonic time: " ChipLogFormatX64 " msec",
-                        sessionHandle->GetSessionTypeString(), &preparedMessage, preparedMessage.GetMessageCounter(),
-                        ChipLogValueX64(kUndefinedNodeId),
-                        ChipLogValueX64(System::SystemClock().GetMonotonicMilliseconds64().count()));
+        char addressStr[Transport::PeerAddress::kMaxToStringSize];
+        destination->ToString(addressStr, Transport::PeerAddress::kMaxToStringSize);
```

Comment:
```suggestion
        destination->ToString(addressStr);
```
so the compiler will determine the size and we won't have mismatches.